test/integration-tests/Uniswap.test.ts

```diff
@@ -1196,6 +1206,480 @@ describe('Uniswap V2 and V3 Tests:', () => {
     })
   })
 
+  describe('Migrator', () => {
+    beforeEach(async () => {
+      // Bob max-approves the v3PM to access his USDC and WETH
+      await usdcContract.connect(bob).approve(v3NFTPositionManager.address, MAX_UINT)
+      await wethContract.connect(bob).approve(v3NFTPositionManager.address, MAX_UINT)
+
+      let bobUSDCBalanceBefore = await usdcContract.balanceOf(bob.address)
+      let bobWETHBalanceBefore = await wethContract.balanceOf(bob.address)
+
+      // need to mint the nft to bob
+      const tx = await v3NFTPositionManager.mint({
+        token0: USDC.address,
+        token1: WETH.address,
+        fee: FeeAmount.LOW,
+        tickLower: 0,
+        tickUpper: 194980,
+        amount0Desired: expandTo6DecimalsBN(2500),
+        amount1Desired: expandTo18DecimalsBN(1),
+        amount0Min: 0,
+        amount1Min: 0,
+        recipient: bob.address,
+        deadline: MAX_UINT,
+      })
+
+      let bobUSDCBalanceAfter = await usdcContract.balanceOf(bob.address)
+      let bobWETHBalanceAfter = await wethContract.balanceOf(bob.address)
+
+      let usdcSpent = bobUSDCBalanceBefore.sub(bobUSDCBalanceAfter)
+      let wethSpent = bobWETHBalanceBefore.sub(bobWETHBalanceAfter)
+
+      // check that the USDC and WETH were spent
+      expect(usdcSpent).to.be.gte(0)
+      expect(wethSpent).to.be.gte(0)
```

Comment:
this doesnt actually check that any usdc was spent? because both can be 0.

I think you need to check that _at least_ one of them is strictly greater than 0. So just 
`expect(usdcSpent > 0 || wethSpent > 0)`